test/stdlib/OSLogPrototypeExecTest.swift

```diff
@@ -66,4 +66,361 @@ if #available(OSX 10.12, iOS 10.0, watchOS 3.0, tvOS 10.0, *) {
     h.log("Process failed after 99% completion")
     h.log("Double percents: %%")
   }
+
+  // A stress test that checks whether the log APIs handle messages with more
+  // than 48 interpolated expressions. Interpolated expressions beyond this
+  // limit must be ignored.
+  OSLogTestSuite.test("messages with too many arguments") {
```

Comment:
These tests that don't call any `expectXXXX` fucntions, they can't really fail, can they? Is it possible to somehow validate the output produced by calling os_log? Maybe it can be done in a separate file that does not use StdlibUnitttest, and instead simply validates the stdout output... Just a thought. What do you think?

Reply:
Yeah, that's is right. We need to have some way of checking the logged messages. I will see how this could be done. As you say, we can't use StdlibUnittest. The logs don't get printed to stdout by default  and may require registering a stream handler. I will create a new PR for that.